### DISCUSSION TITLE
How I disabling and enabling the select component base on checkbox value when each time it clicked.

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#5730] [Help] [answered]
Hi All,

How I disabling and enabling the select component base on checkbox value when each time it clicked.

![image](https://user-images.githubusercontent.com/6214964/219878180-30f0588b-1130-4521-a903-0e083234c7f2.png)

```
      Checkbox::make('is_warehousestaff')->label('Warehouse Staff')->inline()
          ->columnSpan(1)
          ->reactive(),

      Select::make('warehouse_id')
          ->reactive()
          ->disabled(function ($get) {
              ((bool) $get('is_warehousestaff') == true);
          })
          ->columnSpan(1)
          ->required()
          ->label('')
          ->relationship('warehouse', 'nama')
```

This code is not working, select component still enable even the checkbox value is false.

**Comment by zepfietje** [comment, marked as answer]
https://github.com/filamentphp/filament/issues/3068#issuecomment-1213156378
